target/linux/ath79/dts/qca9558_compex_wpj558-16m.dts

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "compex,wpj558-16m", "qca,qca9558";
```

Comment:
This is a dead link.
The 16m suffix should only be kept if there is an _official_ version with different flash size as well.

Reply:
The board was officially sold with a 8 MiB NOR (16 MiB was optional) however this version is not available on Compex website anymore.
I prefer keeping the 16 MiB reference because:
1) the device title in 19.07 specify the 16M version
2) the compex_wpj344-16m & compex_wpj531-16m board definition available in the same file follow the same principle
3) OpenWRT wiki talks about the old version

Reply:
> The board was officially sold with a 8 MiB NOR (16 MiB was optional) however this version is not available on Compex website anymore.

That's a valid argument.

> I prefer keeping the 16 MiB reference because:
> 
> 1. the device title in 19.07 specifies the 16M version

That's not.

> 2. the compex_wpj344-16m & compex_wpj531-16m board definition available in the same file follow the same principle

That's not.

> 3. OpenWRT wiki talks about the old version

Well, what does it tell.

Okay, jokes aside. Due to the one valid reason, you should keep the -16m suffix.